.eslintrc.js

```diff
@@ -46,7 +46,6 @@ module.exports = {
         '@typescript-eslint/no-unused-expressions': ['error'],
         '@typescript-eslint/no-useless-constructor': ['error'],
         '@typescript-eslint/object-curly-spacing': ['error', 'always'],
-        '@typescript-eslint/parameter-properties': ['error'],
```

Comment:
Does the generated code have a problem with this setting? If that's the case, could we change the eslint rule only in that scope?